torch/distributed/_shard/sharded_tensor/metadata.py

```diff
@@ -60,6 +65,7 @@ def __setstate__(
 
     @staticmethod
     def create_from_tensor(tensor: torch.Tensor) -> "TensorProperties":
+        """Create a shared tensor from :class:`torch.Tensor`."""
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.